src/compileNormal.js

```diff
@@ -40,6 +40,10 @@ module.exports = async (options, metaOutput, isInitial, srcFiles) => {
             ...spritesmithResult.properties
         };
 
-        return {sprites, spritesheet};
+        const spritesheet_info = options.apiOptions.spritesheet_info || {
+            name:'spritesheet'
```

Comment:
Default behaviour should not change. I haven't checked it, but I think default value was 'icons'.

Reply:
Oops. My mistake. Your default is correct.